apiserver/facades/client/usermanager/usermanager_test.go

```diff
@@ -821,3 +821,88 @@ func (s *userManagerSuite) TestRemoveUserBulkSharedModels(c *gc.C) {
 	c.Assert(alice.IsDeleted(), jc.IsTrue)
 
 }
+
+func (s *userManagerSuite) TestResetPassword(c *gc.C) {
```

Comment:
We are also missing tests of
a) Resetting more than one user's password successfully in a single call
b) What happens if you call Reset Password when you don't have a password. (This is arguably part of the State level calls, but I think it matters a lot for users, so it would be nice to have a contract here, IMO)